cmd/kola/qemuexec.go

```diff
@@ -36,12 +36,15 @@ var (
 
 	memory  int
 	usernet bool
+
+	ignition string
 )
 
 func init() {
 	root.AddCommand(cmdQemuExec)
 	cmdQemuExec.Flags().BoolVarP(&usernet, "usernet", "U", false, "Enable usermode networking")
 	cmdQemuExec.Flags().IntVarP(&memory, "memory", "m", 0, "Memory in MB")
+	cmdQemuExec.Flags().StringVarP(&ignition, "ignition", "i", "", "Path to ignition config")
```

Comment:
Notably spawn uses userdata (partially because CL had to support cloudinit as well; https://github.com/coreos/mantle/blob/master/cmd/kola/spawn.go#L60)